### PR TITLE
Feature/users index

### PR DESCRIPTION
**Pull request by navase**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ const App = () => {
                 // authenticated={authenticated}
               />
               <Route exact path="/signup" component={signup} />
+              <Route exact path="/users" component={users} />
```
Comment (timstokman) [inline src/App.tsx:34]: /host/users of /chain/users?

File: src/pages/users.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+interface IUsersTableState {
+  users: IUser[] | null;
+}
+
+interface IUserProps {
+  user: IUser;
+}
+
+function Users() {
```
Comment (timstokman) [inline src/pages/users.tsx:20]: As discussed earlier, don't too much unnecessarily

**Comment by pim0223** [comment]
Why was this closed?

**Comment by navase** [comment]
It was only used to show some code to @timstokman, I'm still working on it and will open it again when it's done!

**Pull request closed without merging**